File: Baekjoon/Swift/BOJ_PS/BOJ_PS/Implementation/Q2476-diceGame.swift
//
//  Q2476-diceGame.swift
//  BOJ_PS
//
//  Created by thoonk on 2022/07/31.
//  주사위 게임 / 구현, 수학

var result = 0

for _ in 0..<Int(readLine()!)! {
    let nums = readLine()!.split(separator: " ").map { Int(String($0))! }
    
    if nums[0] == nums[1]
        && nums[1] == nums[2]
        && nums[0] == nums[2] {
        result = max(result, 10000 + nums[0]  * 1000)
    } else if nums[0] == nums[1] {
        result = max(result, 1000 + nums[0] * 100)
    } else if nums[1] == nums[2] {
        result = max(result, 1000 + nums[1] * 100)
    } else if nums[0] == nums[2] {
        result = max(result, 1000 + nums[0] * 100)
    } else {
        result = max(result, nums.max()! * 100)
    }
}

print(result)

File: Baekjoon/Python/BOJ_PS/Implementation/Q2476-diceGame.py
# 주사위 게임 / 구현, 수학
result = 0

for i in range(int(input())):
    a, b, c = map(int, input().split())

    if a == b == c:
        result = max(result, 10000 + a * 1000)
    elif a == b:
        result = max(result, 1000 + a * 100)
    elif b == c:
        result = max(result, 1000 + b * 100)
    elif c == a:
        result = max(result, 1000 + c * 100)
    else:
        result = max(result, max(a, b, c) * 100)

print(result)